### PR TITLE
Document SHOW CREATE INDEX command

### LINKED ISSUE #884: Document: SHOW CREATE INDEX
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/9839

### Which part(s) of the docs might be affected or should be updated? And how?

Add new SQL command `SHOW CREATE INDEX`

### Reference

_No response_

### PR DESCRIPTION
**Pull request by emile-00**
## Info
- **Description**: 
Document SHOW CREATE INDEX command

- **Preview**: 
https://pr-913.d2fbku9n2b6wde.amplifyapp.com/docs/upcoming/sql-show-create-index/

- **Related code PR**: 
https://github.com/risingwavelabs/risingwave/pull/9839

- **Related doc issue**: 
Resolves https://github.com/risingwavelabs/risingwave-docs/issues/884

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/sql/commands/sql-show-create-index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+|Parameter    | Description|
+|---------------|------------|
+|*index_name* |The index to show the query of.|
+
+## Example
+
+We can create a table `t3` and an index `idx1` on `t3`.
+
+```sql
+CREATE TABLE IS NOT EXISTS t3 (
```
Comment (chenzl25) [inline docs/sql/commands/sql-show-create-index.md]:
```suggestion
CREATE TABLE IF NOT EXISTS t3 (
```

File: docs/sql/commands/sql-show-create-index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    v1 int, 
+    v2 int, 
+    v3 int) append only;
```
Comment (chenzl25) [inline docs/sql/commands/sql-show-create-index.md]:
```suggestion
    v3 int);
```

**Review by chenzl25 — CHANGES_REQUESTED** [review]
Append only is an experimental feature, so I think we should not use it in the example.

File: docs/sql/commands/sql-show-create-index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+```
+    Name     |           Create Sql            
+-------------+---------------------------------
+ public.idx1 | CREATE INDEX idx1 ON t3(v1, v2)
+(1 row)
+```
+
+## Related topics
+
+[CREATE INDEX](/sql-create-index.md)
```
Comment (CharlieSYH) [inline docs/sql/commands/sql-show-create-index.md]:
```suggestion
[CREATE INDEX](sql-create-index.md)
```

**Review by CharlieSYH — APPROVED** [review]
Else LGTM

**Review by hengm3467 — APPROVED** [review]

**Review by chenzl25 — APPROVED** [review]

**Pull request merged**